src/logic/operate.js

```diff
@@ -0,0 +1,31 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
@ZabihullahNooriWardak Great work on your app, however, it's not good practice disabling linter checks, kindly consider trying to solve the linter error before disabling linters.